test/python/algorithms/state_fidelities/test_compute_uncompute.py

```diff
@@ -0,0 +1,199 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for Fidelity."""
+
+import unittest
+
+import numpy as np
+
+from qiskit.circuit import QuantumCircuit, ParameterVector
+from qiskit.primitives import Sampler
+from qiskit.algorithms.state_fidelities import ComputeUncompute
+from qiskit.test import QiskitTestCase
+from qiskit import QiskitError
+
+
+class TestComputeUncompute(QiskitTestCase):
+    """Test Compute-Uncompute Fidelity class"""
+
+    def setUp(self):
+        super().setUp()
+        parameters = ParameterVector("x", 2)
+
+        rx_rotations = QuantumCircuit(2)
+        rx_rotations.rx(parameters[0], 0)
+        rx_rotations.rx(parameters[1], 1)
+
+        ry_rotations = QuantumCircuit(2)
+        ry_rotations.ry(parameters[0], 0)
+        ry_rotations.ry(parameters[1], 1)
+
+        plus = QuantumCircuit(2)
+        plus.h(0)
+        plus.h(1)
```

Comment:
This is just a comment, nothing you have to change. You could also write this as 
```suggestion
        plus.h([0, 1])
```
to be more concise 🙂